app/src/routes.js

```diff
@@ -45,7 +45,7 @@ export default (store) => {
         session: () => import(`./components/common/TmSessionSignIn`),
       },
       meta: {
-        feature: "Session",
+        feature: "session",
```

Comment:
ur aware that on PageFeatureNotAvailable now the message shows `session` instead of `Session`?

Reply:
nope. we never actually used `meta` in the message.

https://github.com/luniehq/lunie/pull/4488/files/70cff90e8b23677f6a14523fd4baa0a1fb007ff5#diff-fa9c8bee29345ffb1ef448a7fd064882R9

Reply:
This is easily "checkable"

Reply:
Yes, it doesn't appear:

![image](https://user-images.githubusercontent.com/40721795/87327796-d625e380-c534-11ea-961a-e8c419f95ba2.png)

Reply:
ok, then we removed it at some point